src/commander.c

```diff
@@ -630,7 +630,7 @@ static int commander_process_sign(const char *command)
         char hash[64 + 1];
         char keypath[128];
 
-        if (arr_len < sizeof(hash) + sizeof(keypath) || arr_len > COMMANDER_ARRAY_MAX) {
+        if (arr_len < sizeof(hash) + 1 || arr_len > COMMANDER_ARRAY_MAX) {
```

Comment:
`sizeof(keypath)` always results in 128. This seems to be wrong. The key path size is the the rest of the buffer `(hash - rest_of(arr_len))`. Checking for a min-size of 1.